app/views/ballots/index.html.haml

```diff
@@ -1,5 +1,10 @@
 %h1 Poll details
 -poll = Poll.last
+.voted_img
```

Comment:
This whole template is actually gone now so you might have to implement this different.